src/DeterministicVI.jl

```diff
@@ -16,16 +16,23 @@ export ElboArgs
 
 """
 ElboArgs stores the arguments needed to evaluate the variational objective
-function
+function.
 """
 type ElboArgs{NumType <: Number}
     S::Int64
     N::Int64
+
+    # The number of components in the point spread function.
+    psf_K::Int64
     images::Vector{TiledImage}
     vp::VariationalParams{NumType}
     tile_source_map::Vector{Matrix{Vector{Int}}}
     patches::Matrix{SkyPatch}
     active_sources::Vector{Int}
+
+    # The number of standard deviations away below which a bivariate normal
+    # will be calculated.  Set to Inf to evaluate bivariate normals at all points.
+    num_allowed_sd::Float64
```

Comment:
In theory it results in a discontinuous objective function.  In practice everything works fine even with pretty severe truncations (e.g. two standard deviations).  Above, e.g., 5 standard deviations, it becomes numerically continuous since the contribution of the remaining pixels are below the tolerance of floating point arithmetic.

Anyhow, this kind of concern is why I allowed the user to specify it and set it to ```Inf```.